### PR TITLE
0.8.2

### PR DESCRIPTION
**Pull request by Tezirg**
- Fixes action scope warnings for np.array containing `NaN`
 - `PlotGrid` displays double lines correctly
 -  Upversion `challenge` tensorflow dependency to `2.2.0`
 - Update doc for `make`/`make_new`/`make_old` 
 - Suppress doc warning for member var documented twice

### CODE REVIEW

File: docs/makeenv.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ To get started with such an environment, you can simply do:
 ..code-block:: python
 
     import grid2op
-    env = grid2op.make()
+    env = grid2op.make("rte_case14_realistic")
```
Comment (BDonnot) [inline docs/makeenv.rst:16]: make ne marche plus si on ne lui met rien ? Il faudrait un  paramètre par défaut non ?

**Review by BDonnot — COMMENTED** [review]
Just one clarification on make with no arguments provided would be nice :-)

File: docs/makeenv.rst (hunk 1 above)
Comment (Tezirg) [reply docs/makeenv.rst:16]: Il ya toujours le param par defaut: `def make(dataset="rte_case14_realistic", test=False, **kwargs):`
J'ai juste voulu etre explicite dans la doc ^^

**Review by BDonnot — APPROVED** [review]

File: docs/makeenv.rst [hunk 2]
```diff
@@ -40,17 +38,20 @@ It has the following behavior:
...
-   is why a warning is sent in this case. Also, keep in mind that if you don't pass ``local=True`` then you will not
+   is why a warning is sent in this case. Also, keep in mind that if you don't pass ``test=True`` then you will not
    have the possibility to search for these environments provided in the package.
-5) if nothing is found, :func:`make_new` throws a EnvError.
+6) if no valid environment is found, :func:`make` throws a EnvError.
 
-TODO: explain behaviour of ".grid2opconfig.json"
+Changing cache location
+-------------------------------
+Editing the file ``~/.grid2opconfig.json`` allows you to change the data cache location.
```
Comment (BDonnot) [inline docs/makeenv.rst]: you can do that informatically, with the function `change_local_dir`
Also maybe mention `get_current_local_dir` that get the cache, `list_available_remote_env` and `list_available_local_env` to list the environment already dowloaded.

**Review by BDonnot — COMMENTED** [review]
Maybe a bit more detail in the doc of makeenv, that i didn't see earlier :-/ Beside that everything looks good.

File: docs/makeenv.rst (hunk 2 above)
Comment (Tezirg) [reply docs/makeenv.rst]: Improved section

**Review by BDonnot — APPROVED** [review]

**Pull request merged**